sepal_ui/mapping/map_btn.py

```diff
@@ -7,20 +7,27 @@
 class MapBtn(v.Btn, sw.SepalWidget):
     """
     Btn specifically design to be displayed on a map. It matches all the characteristics of
-    the classic leaflet btn but as they are from ipyvuetify we can use them in combination with Menu to produce on-the-map. The MapBtn is responsive to theme changes.
-    Tiles. It only accept icon as children as the space is very limited.
+    the classic leaflet btn but as they are from ipyvuetify we can use them in combination with Menu to produce on-the-map tiles.
+    The MapBtn is responsive to theme changes. It only accept icon or 3 letters as children as the space is very limited.
+    If both ``logo`` and ``msg`` are filled, msg will have the priority.
 
     Args:
-        logo (str): a fas/mdi fully qualified name
+        logo (str, optional): a fas/mdi fully qualified name.
+        name (str, optional): the name of the btn using 3 letters max.
     """
 
     logo = None
-    "(sw.Icon): a sw.Icon"
+    "(sw.Icon|str): the content of the btn"
 
-    def __init__(self, logo, **kwargs):
+    def __init__(self, logo=None, msg=None, **kwargs):
 
         # create the icon
-        self.logo = sw.Icon(small=True, children=[logo])
```

Comment:
Can we receive "`icon`" key instead of logo? and later rename if necessary to whatever else?

Reply:
unfortunately we cannot use "icon" as it's a parameter of vuetify btn (https://vuetifyjs.com/en/api/v-btn/#props-icon) and we decided at some point not to override existing keys for something else. 

So I'm happy to change the name if  you have any other idea that is not part of the existing keys of the vuetify Btn component

Reply:
Ok, we have to follow our own rules. We have replaced `type` by `type_` in some elements, what do you think about `icon_`?